test/unit/typography/font_loading.js

```diff
@@ -59,7 +59,7 @@ suite('Fonts outside preload()', function () {
 
     test('should call success-callback with object when font loads',
       function (done) {
-        p5Inst.loadFont('manual-test-examples/p5.Font/acmesa.ttf',
+        p5Inst.loadFont('../manual-test-examples/p5.Font/acmesa.ttf',
```

Comment:
sorry, I wasn't clear. I was asking about the idea of setting a working directory for all unit tests to be run from. the solution in the changes you pushed recently only sets the directory for this particular file, but leaves in the "../" in other files still.